src/sound/Sound.js

```diff
@@ -174,6 +174,43 @@ export class Sound extends PsychObject
 		};
 	}
 
+	/**
+	 * Set value for a particular player
+	 *
+	 * @param {number|string} [value = "C"] the sound value (see above for a full description)
+     */
+	setValue(value = "C", log = false)
+	{
+		this._setAttribute("value", value, log);
+
+		if (this._player === undefined)
+		{
+			return;
+		}
+
+		// Analyse the value and change the type of player if needed.
+		const valSupported = this._player.constructor.checkValueSupport(value);
```

Comment:
Not working fully yet. Issues arise when trying to convert from one type of player to the other and passing note as a string.